File: official/guides/getting-started/php/create_from_address.php
require_once("path/to/lib/easypost.php");
\EasyPost\EasyPost::setApiKey("<YOUR_TEST/PRODUCTION_API_KEY>");

$fromAddress = \EasyPost\Address::create(array(
  'company' => 'EasyPost',
  'street1' => '417 Montgomery Street',
  'street2' => '5th Floor',
  'city' => 'San Francisco',
  'state' => 'CA',
  'zip' => '94104',
  'phone' => '415-528-7555'
));

File: official/guides/getting-started/python/create_from_address.py
import easypost

easypost.api_key = "<YOUR_TEST/PRODUCTION_API_KEY>"

fromAddress = easypost.Address.create(
    company="EasyPost",
    street1="417 Montgomery Street",
    street2="5th Floor",
    city="San Francisco",
    state="CA",
    zip="94104",
    phone="415-528-7555",
)
